services/ops/FailoverAgent/README.md

```diff
@@ -0,0 +1,64 @@
+## Failover Agent
+
+The Failover agent provides a generic high availability option to VOLTTRON.  When the primary platform becomes 
+inactive the secondary platform will start an installed agent.
+
+
+### Standard Failover
+
+There are two behavior patterns implemented in the agent. In the default
+configuration, the secondary instance will ask Volttron Central to verify
+that the primary instance is down. This helps to avoid a split brain scenario.
+If neither Volttron Central nor the other Failover instance is reachable
+then the Failover agent will stop the agent it is managing. These states are
+shown in the tables below.
+
+**Primary Behavior**
+
+|                 | VC Up | VC Down |
+|-----------------|-------|---------|
+| Secondary Up    | start | start   |
+| Secondary Down  | start | stop    |
+
+**Secondary Behavior**
+
+|              | VC Up           | VC Down |
+|--------------|-----------------|---------|
+| Primary Up   | stop            | stop    |
+| Primary Down | Verify with VC before starting | stop    |
```

Comment:
I find this description difficult to follow. Are the two behavior patterns something I choose between, or are we saying that in the "standard failover" mode of operation, the agent is able to follow two different patterns each as needed?

A diagram may be helpful but also not easy in markdown. Otherwise I think it would be helpful to be very explicit about: "We have these platforms with (at least) these agents " (and be careful/precise about the names). In particular, is the failover on the same platform as the primary? as the secondary?... vc on the primary the secondary? multiple options? does this only work with vc-based multi-plaform? How does the Failover agent decide if the primary is up/down?